### DISCUSSION TITLE
Bind `this` In `x-init` To Node

### DISCUSSION THREAD

**Discussion by fweth** [alpinejs/alpine#2645] [3. Feature ideas]
Hi, would it be hard to bind `this` in `x-init` to the node on which the `x-init` is placed?  So that we can e.g. place event listeners on the node without having to create an `x-ref` first?

**Comment by Tim-Wils** [comment]
You can use `$el` to access the element ( https://alpinejs.dev/magics/el ) :)

**Comment by fweth** [comment]
Hey, that's super cool, thanks a lot!
